lib/storage/metadata/mongoclient/MongoClientInterface.js

```diff
@@ -1033,6 +1033,7 @@ class MongoClientInterface {
         const masterKey = formatMasterKey(objName, vFormat);
         MongoUtils.serialize(objVal);
         // eslint-disable-next-line
+        objVal.originOp = 's3:ObjectRemoved:Delete'
```

Comment:
good catch !
Yes the line is needed, we get the following warning otherwise
`1035:9  warning  Assignment to property of function parameter 'objVal'  no-param-reassign`